### DISCUSSION TITLE
Nuget signing with azure trusted signing accounts

### DISCUSSION THREAD

**Discussion by jmecosta** [NuGet/Home#13458] [General]
Hi,

ive looked around and i cant find any references on how to sign nuget packages using the new azure trusted account service. The service is fully functional for signing executable files.

Any pointers where i could start to get the support for it? 

thanks 
Jorge Costa

**Comment by MichaelGrafnetter** [comment]
The support for Azure Trusted Signing has actually been implemented in the `sign` global tool, see https://github.com/dotnet/sign/issues/683 .  Still beta/preview.

**Comment by jmecosta** [comment]
Yes at the moment only thing missing is nuget gallery support for short live cer files which is handled in another proposal
